packages/core/src/utils/tenant.ts

```diff
@@ -56,16 +60,16 @@ const getTenantIdFromCustomDomain = async (
   const cachedValue = await trySafe(async () => redisCache.get(getDomainCacheKey(url)));
 
   if (cachedValue) {
-    return cachedValue === notFoundValue ? undefined : cachedValue;
+    return cachedValue;
   }
 
   const { findActiveDomain } = createDomainsQueries(pool);
 
   const domain = await findActiveDomain(url.hostname);
 
-  await trySafe(async () =>
-    redisCache.set(getDomainCacheKey(url), domain?.tenantId ?? notFoundValue, 60)
-  );
+  if (domain?.tenantId) {
+    await trySafe(async () => redisCache.set(getDomainCacheKey(url), domain.tenantId));
```

Comment:
No expire time specified?

Reply:
The default value is 30 mins. Because we will now refresh it when domain status changed, 30 mins is OK